Makefile

```diff
@@ -25,8 +28,17 @@ var/lib/grafana/%.json: src/%.jsonnet
 	mkdir -p $(shell dirname $@)
 	bin/jsonnet -o /$@ $<
 
+var/lib/grafana/plugins:
+	mkdir -p ./var/lib/grafana/plugins
+
+var/lib/grafana/plugins/grafana-groupedbarchart-panel-master:
+	curl -Lo /tmp/grafana_bar_chart_plugin.zip $(BAR_CHART_PLUGIN)
+	rm -rf ./var/lib/grafana/plugins/*
```

Comment:
Maybe we should add this to the `clean` rule, WDYT?

Reply:
Nice catch) Thanks) Indeed, this should not be here)

Reply:
Added the `clean` rule which removes the `var` directory)